### ISSUE TITLE
errors is undefined in subelements.py

### ISSUE THREAD

**Issue by pbowditch** [ansible/ansible#11624]
Issue Type:

Bug Report

Ansible Configuration:

[defaults]
command_warnings = False
forks            = 64
gathering        = smart
nocows           = 1
pattern          = *
remote_tmp       = $HOME/.ansible/tmp
transport        = smart
inventory        = inventory
hash_behaviour   = merge

Ansible Version:

ansible 2.0.0 (devel ba7243c5f9) last updated 2015/07/16 15:01:11 (GMT +100)
  lib/ansible/modules/core: (detached HEAD 291fef3b34) last updated 2015/07/16 15:01:36 (GMT +100)
  lib/ansible/modules/extras: (detached HEAD ff2386faf4) last updated 2015/07/16 15:01:40 (GMT +100)
  v1/ansible/modules/core: (detached HEAD f8d8af17cd) last updated 2015/07/16 15:01:45 (GMT +100)
  v1/ansible/modules/extras: (detached HEAD 495ad450e5) last updated 2015/07/16 15:01:49 (GMT +100)
  configured module search path = None

Environment:

CentOS 6

Summary:

Python stack trace received when running an incorrectly configured playbook.  Expected to see the human readable error message.

Steps To Reproduce:

Trying to iterate over a dictionary using with_subelements rather than a list.  

```
with_subelements:
  - my_dict
  - subfield_also_a_dict
```

Expected Results:

``` python
fatal: [localhost]: FAILED! => {
    "failed": true,
    "msg": "ERROR! the key details should point to a list, got '{u'description': u'', u'vars': None}'"
}
```

Actual Results:

``` python
An exception occurred during task execution. The full traceback is:
Traceback (most recent call last):
  File "/export/home/developer/ansible/lib/ansible/executor/process/worker.py", line 118, in run
    executor_result = TaskExecutor(host, task, job_vars, new_connection_info, self._new_stdin, self._loader, shared_loader_obj).run()
  File "/export/home/developer/ansible/lib/ansible/executor/task_executor.py", line 80, in run
    items = self._get_loop_items()
  File "/export/home/developer/ansible/lib/ansible/executor/task_executor.py", line 134, in _get_loop_items
    items = lookup_loader.get(self._task.loop, loader=self._loader).run(terms=loop_terms, variables=self._job_vars)
  File "/export/home/developer/ansible/lib/ansible/plugins/lookup/subelements.py", line 96, in run
    raise errors.AnsibleError("the key %s should point to a list, got '%s'" % (subkey, subvalue[subkey]))
NameError: global name 'errors' is not defined

fatal: [localhost]: FAILED! => {
    "failed": true,
    "stdout": ""
}
```

I believe the issue is the "from ansible.errors import *" statement in subelements.py which can be replaced with "from ansible import errors"

**Comment by srgvg** [comment]
As to be in line with other plugins, I'm fixing this by changing `raise errors.AnsibleError` to `raise AnsibleError`

@pbowditch Can you verify this is fixed with #11628?

**Comment by pbowditch** [comment]
I can confirm this is fixed.  Thanks
